src/wallet/rpcwallet.cpp

```diff
@@ -1145,6 +1145,9 @@ UniValue sendmany(const JSONRPCRequest& request)
     if (totalAmount > nBalance)
         throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, "Account has insufficient funds");
 
+    // Shuffle recipient list
+    std::random_shuffle(vecSend.begin(), vecSend.end(), GetRandInt);
```

Comment:
Please rebase on top of #12742 and use `std::shuffle` from cpp11 to avoid future code churn.